app/pathogen/arbovirus/dashboard/ArboCountryPopUpContent.tsx

```diff
@@ -0,0 +1,49 @@
+import { typedGroupBy } from "@/lib/utils"
+import { ShortformArbovirus, convertArboSFtoArbo } from "../analyze/recharts"
+import { PopUpContentRow } from "./ArboStudyPopupContent";
+
+interface ArboCountryPopupContentProps {
+  record: {
+    id: string,
+    alpha3CountryCode: string,
+    countryName: string,
+    latitude: string,
+    longitude: string,
+    dataPoints: { pathogen: ShortformArbovirus }[],
+  }
+}
+
+export function ArboCountryPopupContent({ record }: ArboCountryPopupContentProps): React.ReactNode {
+  const dataPointsGroupedByArbovirus = typedGroupBy(record.dataPoints, (dataPoint) => dataPoint.pathogen);
+
+  return (
+    <div className="w-[460px] bg-white pt-2">
```

Comment:
There's a bit of copy-pastel from `ArboStudyPopUpContent` like the maximum width, the paddings, stuff like that. Probably worth consolidating some of that stuff into components, just not right now.